core/src/main/java/com/amplifyframework/logging/AndroidLoggingPlugin.java

```diff
@@ -66,6 +77,7 @@ public String getPluginKey() {
     public void configure(
             JSONObject pluginConfiguration,
             @NonNull Context context) {
+        storeLogs = (context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE) != 0;
```

Comment:
Can you centralize this check into some helper class? I think you perform this check in a couple places now.
```java
final class IsDebuggable {
    private IsDebuggable() {}

    boolean check(Context context) {
        return context.getApplicationInfo().flags & ApplicationInfo.FLAG_DEBUGGABLE != 0;
    }
}
```
(You can get creative with the naming, this is just one idea.)